frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Tab-Sidebar-ODC-4116.feature

```diff
@@ -131,3 +149,13 @@ Scenario: Edit Health Checks option from Actions dropdown on Sidebar for Deploym
     And user clicks on Save button
     Then user will be redirected to Topology page
     And user will see Readiness Probe removed on the Add Health Checks page
+
+
+@smoke, @regression
+Scenario: Edit Health Checks option for Helm Chart through Context Menu: MH-06-TC04
+    Given user is at Topology page
+    And user has a helm release with named "node-js-ex"
+    When user right clicks on the "node-js-ex" to open the Context Menu
+    And user clicks on Edit Health Checks
+    Then user sees Readiness Probe already added
```

Comment:
Mention it in given that particular helm release have Readiness and Liveness probe already added

Reply:
It's by default @sanketpathak and it's the test cases of checking automatically added health checks